### PR TITLE
deprecate old csi API endpoints

### PR DESCRIPTION
**Pull request by geoffrey1330**
Tested the changes by running e2e tests.

```
Will run 5 of 5 specs
------------------------------
• [SLOW TEST] [61.578 seconds]
CSI Driver tests Control Plane: delete second lvol while the first lvol has IO running if a node has lvol with IO running, adding and deleting an new lvol from the same node should work
/home/ubuntu/actions-runner/_work/simplyBlockDeploy/simplyBlockDeploy/spdk-csi/e2e/controlplane.go:26
------------------------------
•
------------------------------
• [SLOW TEST] [50.774 seconds]
SPDKCSI-NVMEOF Test SPDK CSI Dynamic Volume Provisioning Test the flow for Dynamic volume provisioning
/home/ubuntu/actions-runner/_work/simplyBlockDeploy/simplyBlockDeploy/spdk-csi/e2e/nvmeof.go:30
------------------------------
• [SLOW TEST] [122.170 seconds]
SPDKCSI-NVMEOF Test SPDK CSI Dynamic Volume Provisioning Test the flow for Caching nodes
/home/ubuntu/actions-runner/_work/simplyBlockDeploy/simplyBlockDeploy/spdk-csi/e2e/nvmeof.go:51
------------------------------
• [SLOW TEST] [80.269 seconds]
SPDKCSI-NVMEOF Test SPDK CSI Dynamic Volume Provisioning Test multiple PVCs
/home/ubuntu/actions-runner/_work/simplyBlockDeploy/simplyBlockDeploy/spdk-csi/e2e/nvmeof.go:79
------------------------------

Ran 5 of 5 Specs in 315.185 seconds
SUCCESS! -- 5 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestE2E (315.19s)
PASS
ok  	github.com/spdk/spdk-csi/e2e	316.544s
```

### CODE REVIEW

**Review by boddumanohar — APPROVED** [review]
Thanks for running the e2e tests. Looks good.

**Comment by geoffrey1330** [comment]
We can't merge yet until @Hamdy-khader merges this [PR](https://github.com/simplyblock-io/sbcli/pull/37) if not the route will break in `sbcli-dev`

**Comment by boddumanohar** [comment]
we still need to deprecate the `/csi` calls in https://github.com/simplyblock-io/spdk-csi/blob/317d9799f9c3773760089fa1aa3e0796ea4adf1d/pkg/util/nvmf.go#L153-L173

**Comment by geoffrey1330** [comment]
> we still need to deprecate the `/csi` calls in
> 
> https://github.com/simplyblock-io/spdk-csi/blob/317d9799f9c3773760089fa1aa3e0796ea4adf1d/pkg/util/nvmf.go#L153-L173

I missed this.

File: pkg/util/nvmf.go
Comment (geoffrey1330) [inline pkg/util/nvmf.go:155]: The changes was done here @boddumanohar

**Pull request merged**
